src/components/Button/Button.st.css

```diff
@@ -189,3 +189,58 @@
     padding: 10px 16px;
     font: fallback(font(value(MainTextFont)), font(value(LargeFont)));
 }
+
+:vars{
+    secondaryBorderWidth: fallback(value(MainBorderWidth), 1px);
```

Comment:
I think you're missing a definition for `primaryBorderWidth`, no?
Or maybe you meant to use `PrimaryBorderWidth` (with capital 'P')?